src/app/shared/store/paginator.state.ts

```diff
@@ -19,14 +19,17 @@ import {
   SetItemsPerPage,
   OnPageChangeHistoryLog,
   SetAchievementsPerPage,
-  OnPageChange
+  OnPageChange,
+  SetReportsPerPage,
+  OnPageChangeReports
 } from './paginator.actions';
 
 export interface PaginatorStateModel {
   achievementPerPage: number;
   workshopsPerPage: number;
   directionsPerPage: number;
   applicationsPerPage: number;
+  reportsPerPage: number;
```

Comment:
I am sure this is redundant. We had an agreement that for all table data, we will use itemsPerPage (as it does for provider-list, user-list, admin-list). Check how this is implemented any of those tables  

PS: it should be renamed to tableItemsPerPage, so it will be more understandable, but it could be added as Tech Debt item